src/components/implementation/no_interface/events_prototype/Makefile

```diff
@@ -3,7 +3,7 @@ ASM_OBJS=
 COMPONENT=rust_test.o
 INTERFACES=
 DEPENDENCIES=
-ADDITIONAL_LIBS=-lcobj_format -lcos_defkernel_api -lcos_kernel_api -lsl -lheap -lsl_thd_static_backend -lsl_lock -lsl_mod_fprr
+ADDITIONAL_LIBS=-lcobj_format -lcos_defkernel_api -lcos_kernel_api -lsl_rawmgr -lsl_sched -lheap -lsl_thd_static_backend -lsl_lock -lsl_mod_fprr
```

Comment:
trying to grok what `rawmgr` means.  Raw -- low-level access.  Mgr -- providing the service?  Is that what you're going for?

Reply:
Raw = low-level raw kernel API, mgr = managing (creation/deletion..) resources.

Reply:
There are 2 sl thd management libraries now, one that uses resource manager and the other that uses kernel API.
Resource manager links to sl_rawmgr as it is managing the raw resources and other higher level schedulers link to sl_resmgr that internally uses resmgr interface. 

This is also a case of component with no resource manager, so this links to sl_rawmgr.

Reply:
The `mgr` is throwing me off, I think.  We are depending only on the kernel, so `mgr` seems strange.   It should be noted that we might be implementing a `mgr` (i.e. provide resource management features to other components), or we might not.

Maybe just `sl_raw` is enough...

Reply:
Kk, I initially had it `sl_raw` then changed it to `sl_rawmgr`. 
I'll change it back to `sl_raw`.